docs/products/kafka/howto/provectus-kafka-ui.rst

```diff
@@ -0,0 +1,50 @@
+Use Provectus® UI for Apache Kafka® with Aiven for Apache Kafka®
```

Comment:
Can't find anything on Provectus's website about their trademark, so I guess the "r" is fine.